### PR TITLE
#2009 Change admin/RA dashboard graphs to show onboarded or not

### PR DESCRIPTION
**Pull request by stenington**
Fix for #2009, follows in the footsteps of #1915 to simplify the student/mentor onboarding pie charts.

### CODE REVIEW

File: app/models/data_analyses/onboarding_mentors_data_analysis.rb [hunk 1]
```diff
@@ -12,10 +12,27 @@ def init_data
           .in_region(user)
       end
 
-      @cleared_mentors = @mentors.bg_check_clear.consent_signed
-      @signed_consent_mentors = @mentors.bg_check_submitted.consent_signed
-      @cleared_bg_check_mentors = @mentors.bg_check_clear.consent_not_signed
-      @neither_mentors = @mentors.bg_check_unsubmitted.consent_not_signed
+      #FIXME: these queries were cribbed from the datagrid, but should be shared somehow
```
Comment (stenington) [inline app/models/data_analyses/onboarding_mentors_data_analysis.rb]: This is the reason this is marked as **WIP**.

@adrichey My inclination here is to move the queries you added in #1915 to the accounts datagrid into scopes on Account. I know there will be duplication between those scopes and the scopes in e.g. MentorProfile, but I'm thinking of maybe trying to address that with tests? I'm not sure if there's a better way to share scopes between the two models, but I can test that both scopes give me the same users in a variety of scenarios.

So basically there would be `Account.onboarded_mentors` and `MentorProfile.onboarded`, and tests would help ensure those queries match, and then whichever is most appropriate can be used in the accounts datagrid and in these data analyses classes and we can expect the same results from both. (Repeat for students.)

What do you think about that?

Comment (adrichey) [reply app/models/data_analyses/onboarding_mentors_data_analysis.rb]: We already have quite a bit of duplication on the Account models that also falls on the mentor and student profile models, so I am not particularly turned off by the duplication, provided the tests are in place. I think that sounds reasonable.

**Comment by stenington** [comment]
Some notes for review:

This PR tries to standardize the definition of `onboarded` for a mentor a bit. It does away with the `MentorProfile.onboarded` scope, which as best I can tell was used only in tests. Then it adds `Account.onboarded_mentors` and `Account.onboarding_mentors` scopes that try to match the logic in `MentorProfile.onboarded?` in terms of requirements.

For the dashboard graphs, it does away with the `international` concept which was used to display graphs including or excluding background checks, depending on location. That's now buried inside the concept of "onboarded" so both international and US RAs can see the same mentor graph.

For students, we're switching from the notion of "permitted" to the notion of "onboarded", so there's a whole bunch of renaming of things to follow suit.
